code/_onclick/other_mobs.dm

```diff
@@ -32,6 +32,8 @@
 				SPAN_DANGER("\The [user] smashes through \the [src]!"),
 				SPAN_DANGER("You smash through \the [src]!")
 			)
+			if (MUTATION_FERAL in user.mutations)
+				qdel(src)
```

Comment:
I'm not sure I like this qdel. Is it necessary? Shouldn't the atoms decide how they handle destruction if their health reaches 0? Unless I'm reading this proc wrong..

Reply:
It does seem quite goofy, blame upstream.